src/core/optimization_container.jl

```diff
@@ -638,31 +630,37 @@ function check_optimization_container(optimization_container::OptimizationContai
     return
 end
 
-function _build!(
+function get_problem_size(optimization_container::OptimizationContainer)
+    model = optimization_container.JuMPmodel
+    vars = JuMP.num_variables(model)
+    cons = 0
+    for (exp, c_type) in JuMP.list_of_constraint_types(model)
+        cons += JuMP.num_constraints(model, exp, c_type)
+    end
+    return "The current total number of variables is $(vars) and total number of constraints is $(cons)"
+end
+
+function build_impl(
```

Comment:
Did you inadvertently remove the `!` from this function name?

Reply:
yes, this was a silly mistake.

Reply:
I fixed this in the forthcoming PR. I will merge to open a new PR with changes to simulation